paper.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Software Carpentry metagenomic workshop'
+title: 'A Data Carpentry style metagenomic workshop'
```

Comment:
In other places you use "Data Carpentry" and I suggest you remain consistent. I think Data Carpentry is a better fit for your curriculum given that it focuses on a data set and a set of domain-specific skills, while Software Carpentry is more about computing skills.

Adding "style" also emphasizes that this is not an officially supported Carpentries lesson.

Reply:
Thank you ! 👍